config/pickit.ini

```diff
@@ -207,15 +206,15 @@ magic_ogre_gauntlets=0
 ;==================================================================;
 
 ;rare jewelry
-rare_amulet=1
+rare_amulet=1, (2_amazon_skills, 2_all_skills, 2_assasin_skills, 2_barbarian_skills, 2_Druid_skills, 2_necromancer_skills, 2_paladin_skills, 2_sorceress_skills)
 rare_jewel=1
-rare_ring=1
+rare_ring=1, (and(life_stolen, mana_stolen), faster_cast_rate)
```

Comment:
puh. There can be really good stats rings without fcr or ll, ml. Rings don't take up much space, I'd vote to just keep them all.

Reply:
Removed that include filter in this case